README.md

```diff
@@ -60,7 +60,7 @@ dependencies {
 
 How To (Compiling Jar From Source)
 ------
-To compile Paper, you need JDK 8, maven, and an internet connection.
+To compile Paper, we recommend use JDK 11, maven, and an internet connection.
```

Comment:
`we recommend use of`
`we recommend you use`

I would rather this just be the base requirements and we can add an `or newer` but it doesn't matter so much. Fix the grammar either way.

Reply:
This is technically poor wording because "we recommend you use" modifies both Java 11 AND Maven, however, maven is required.

Reply:
then maybe only rename `we recommend use` to `you need`, but if is for grammar why the `maven, and` and not  `maven and`

Reply:
It's an Oxford comma: https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/